NDB.Covid19/NDB.Covid19.Droid/Resources/layout/settings_consents.xml

```diff
@@ -85,44 +75,40 @@
                 android:visibility="gone"
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toBottomOf="@id/consent_info"
-                />
+                app:layout_constraintTop_toBottomOf="@id/consent_info" />
         </androidx.constraintlayout.widget.ConstraintLayout>
     </ScrollView>
 
     <View
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
```

Comment:
0 dp?

Reply:
0 dp in constraint layout means the view is streched between left and right border that are defined by:
app:layout_constraintEnd_toEndOf, app:layout_constraintStart_toStartOf and others